content/tutorial/tutorial.md

```diff
@@ -12,185 +12,189 @@ redirect_from:
+यदि आपको जावास्क्रिप्ट की समीक्षा करने की आवश्यकता है, तो हम [इस मार्गदर्शिका](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) को पढ़ने की सलाह देते हैं। ध्यान दें कि हम ईएस 6 से कुछ सुविधाओं का उपयोग कर रहे हैं - जो की जावास्क्रिप्ट का एक हालिया संस्करण। इस ट्यूटोरियल में, हम [एरो फ़ंक्शंस](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), [क्लासेस](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes), [`let`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) और [`const`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const) स्टेटमेंट्स का उपयोग कर रहे हैं। ईएस 6 कोड क्या संकलित करता है यह जांचने के लिए आप [बैबल REPL](babel://es5-syntax-example) का उपयोग कर सकते हैं।
 
-## Setup for the Tutorial {#setup-for-the-tutorial}
+## ट्यूटोरियल के लिए सेटअप {#setup-for-the-tutorial}
 
-There are two ways to complete this tutorial: you can either write the code in your browser, or you can set up a local development environment on your computer.
+इस ट्यूटोरियल को पूरा करने के दो तरीके हैं: आप या तो अपने ब्राउज़र में कोड लिख सकते हैं, या आप अपने कंप्यूटर पर स्थानीय विकास वातावरण सेट कर सकते हैं।
 
-### Setup Option 1: Write Code in the Browser {#setup-option-1-write-code-in-the-browser}
+### सेटअप विकल्प 1: ब्राउज़र में कोड लिखें {#setup-option-1-write-code-in-the-browser}
 
-This is the quickest way to get started!
+यह आरंभ करने का सबसे तेज़ तरीका है!
 
-First, open this **[Starter Code](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** in a new tab. The new tab should display an empty tic-tac-toe game board and React code. We will be editing the React code in this tutorial.
+सबसे पहले, इस **[प्रारंभक कोड](https://codepen.io/gaearon/pen/oWWQNa?editors=0010)** को एक नए टैब में खोलें। 
+नए टैब में एक खाली टिक-टैक-टो गेम Board और React कोड प्रदर्शित होना चाहिए। हम इस ट्यूटोरियल में React कोड का संपादन करेंगे।
 
-You can now skip the second setup option, and go to the [Overview](#overview) section to get an overview of React.
+अब आप दूसरे सेटअप विकल्प को छोड़ सकते हैं, और React का ओवरव्यू प्राप्त करने के लिए [ओवरव्यू](#overview) अनुभाग पर जाएं।
 
-### Setup Option 2: Local Development Environment {#setup-option-2-local-development-environment}
+### सेटअप विकल्प 2: स्थानीय विकास पर्यावरण {#setup-option-2-local-development-environment}
 
-This is completely optional and not required for this tutorial!
+यह पूरी तरह से वैकल्पिक है और इस ट्यूटोरियल की आवश्यकता नहीं है!
 
 <br>
 
 <details>
 
-<summary><b>Optional: Instructions for following along locally using your preferred text editor</b></summary>
+<summary><b>वैकल्पिक: अपने पसंदीदा पाठ संपादक का उपयोग करके स्थानीय स्तर पर अनुसरण करने के निर्देश</b></summary>
```

Comment:
ऐच्छिक

Reply:
पाठ संपादक -> should not be translated/need simpler words

Reply:
स्थानीय स्तर पर अनुसरण करने के निर्देश -> again complex

Reply:
How about this
`अपने पसंदीदा टेक्स्ट एडिटर का उपयोग करके अपने लोकल स्तर पर निर्देशों का पालन करें`

Reply:
can there be something better for लोकल स्तर? rest is good

Reply:
closest I can think of is this `लोकल एनवायरनमेंट`

Reply:
how about लोकल पर्यावरण?